pkg/types/hashmap/list.go

```diff
@@ -17,15 +17,14 @@ type Item struct {
 type List []Item
 
 // Flatten converts a flow map to a flat table / list
-func (m *Map) Flatten() (v4List List, v6List List) {
-	v4List, v6List = make(List, 0), make(List, 0)
-
-	for i := m.Iter(); i.Next(); {
-		if k := types.Key(i.Key()); k.IsIPv4() {
-			v4List = append(v4List, Item{i.Key(), i.Val()})
-		} else {
-			v6List = append(v6List, Item{i.Key(), i.Val()})
-		}
+func (a AggFlowMap) Flatten() (v4List List, v6List List) {
+	v4List, v6List = make(List, 0, a.V4Map.count), make(List, 0, a.V6Map.count)
+
+	for i := a.V4Map.Iter(); i.Next(); {
+		v4List = append(v4List, Item{i.Key(), i.Val()})
```

Comment:
Is `append` on a pre-sized slice faster than indexing `v4List`? You know how big it is up front.

Reply:
It should (tm) be identical at runtime, but I'll still adapt because indexing is definitively _never_ slower _and_ I find it more readable here...